recipes/libcusolver/build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# Install to conda style directories
+[[ -d lib64 ]] && mv lib64 lib
+mkdir -p ${PREFIX}/lib
+[[ -d pkg-config ]] && mv pkg-config ${PREFIX}/lib/pkgconfig
+[[ -d "$PREFIX/lib/pkgconfig" ]] && sed -E -i "s|cudaroot=.+|cudaroot=$PREFIX|g" $PREFIX/lib/pkgconfig/cusolver*.pc
+
+[[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
+[[ ${target_platform} == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
+[[ ${target_platform} == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
+
+for i in `ls`; do
+    [[ $i == "build_env_setup.sh" ]] && continue
+    [[ $i == "conda_build.sh" ]] && continue
+    [[ $i == "metadata_conda_debug.yaml" ]] && continue
+    if [[ $i == "lib" ]] || [[ $i == "include" ]]; then
+        # Headers and libraries are installed to targetsDir
+        mkdir -p ${PREFIX}/${targetsDir}
+        mkdir -p ${PREFIX}/$i
+        cp -rv $i ${PREFIX}/${targetsDir}
+        if [[ $i == "lib" ]]; then
+            for j in "$i"/*.so*; do
```

Comment:
```suggestion
            for j in "$i"/*.so*; do
            for j in "$i"/{*.so,stubs/*.so}*; do
```
Do we want stubs symlinked from `lib/stubs/*.so` to `targets/{{ target_platform }}/stubs/*.so`?

cudart [tries to](https://github.com/adibbley/staged-recipes/blob/cuda-cudart/recipes/cuda-cudart/meta.yaml#L218)
But libcublas does not

cc @bdice

Reply:
cc: @robertmaynard for input

Reply:
Let's follow the precedent set by libcublas on this (unless we have a reason to change libcublas as well). The cudart package is somewhat different than the math libraries in structure and use of static linking. We had some discussions about the topic of stubs, but I don't recall a conclusive reason for/against this, from our past conversations. We can always add symlinks in `$PREFIX/lib/stubs` later if needed.

Reply:
Thanks, that is what I thought as well. Just wanted to be sure since I couldn't find it explicitly stated anywhere. We can resolve this with no change.

Reply:
I agree with @bdice